ouroboros-network/src/Ouroboros/Network/PeerSelection/PeerSelectionActions.hs

```diff
@@ -5,6 +5,7 @@
 {-# LANGUAGE RankNTypes          #-}
 {-# LANGUAGE ScopedTypeVariables #-}
 {-# LANGUAGE TupleSections       #-}
+{-# LANGUAGE DisambiguateRecordFields #-}
```

Comment:
Why do you need this pragma?

Reply:
PeerSelectionActions and PeerSelectionActionArgs both have peerTargets, which holds the same value, but without the pragma we give them different names. In one of review comments somewhere @coot mentioned it may be a good idea to start experimenting with these pragmas to declutter this proliferation of names.